content/en/docs/concepts/services-networking/_index.md

```diff
@@ -1,63 +1,175 @@
+- Cluster networking provides communication between different Pods.
+- The [Service](/docs/concepts/services-networking/service/) API lets you
+  [expose an application running in Pods](/docs/tutorials/services/connect-applications-service/)
+  to be reachable from outside your cluster.
+  - [Ingress](/docs/concepts/services-networking/ingress/) and [Gateway](https://gateway-api.sigs.k8s.io/) provide
+    extra functionality specifically for exposing your applications, websites and APIs, usually to clients outside
+    the cluster. Ingress and Gateway often use a load balancer to make that work reliably and at scale.
+- You can also use Services to
+  [publish services only for consumption inside your cluster](/docs/concepts/services-networking/service-traffic-policy/).
+
+The [Connecting Applications with Services](/docs/tutorials/services/connect-applications-service/) tutorial lets you learn
+about Services and Kubernetes networking with a hands-on example.
+
+Read on to learn more about the [Kubernetes network model](#kubernetes-network-model).
+
+## Terminology
+
+L2 bridge
+: a (virtual) [layer 2](https://en.wikipedia.org/wiki/Data_link_layer) bridge enabling inter-pod connectivity on the same host.
+
+Encapsulation
+: Ability to encapsulate layer 2 or layer 3 packets belonging to an _inner network_ with an _outer network_ header for
+  transport across the _outer network_.
+  This forms a virtual _tunnel_ where the encapsulation function is performed at tunnel ingress and de-encapsulation
+  function is performed at tunnel egress. You can think of pods on nodes as the inner network and nodes networked
+  together as the outer network. [IPIP](https://www.rfc-editor.org/rfc/rfc2003) and
+  [VXLAN](https://www.rfc-editor.org/rfc/rfc7348) are two examples of encapsulation that network plugins use to support
+  cluster networking.
+
+Network namespace
+: Form of isolation used on Linux, where different processes (such as in containers) see a different set of network
+  interfaces and configuration than the host system. The host system is represented by a root network namespace,
+  which is often what network plugins use to set up connectivity between nodes (and between Pods on those nodes).
 
-Kubernetes imposes the following fundamental requirements on any networking
-implementation (barring any intentional network segmentation policies):
+kube-proxy
+: Part of Kubernetes, `kube-proxy` is optional component that you run on each Node.
+  The kube-proxy ensures that clients can connect to [Services](/docs/concepts/services-networking/service/),
+  including to any backend Pods that make up the Service. Clients might be other  Pods, or they could be connecting from outside the cluster.
+  Some network plugins provide their own alternative to kube-proxy, which means   you don't need to install it when you use that particular plugin.
```

Comment:
OK, but: I'm trying to find the smallest set of changes to this PR so that we can go from there-is-no-diagram to there-is-at-least something.
(Does this feedback block a merge?)

Reply:
```suggestion
  Some network plugins provide their own alternative to kube-proxy.
```